addon/templates/components/common/bs-carousel.hbs

```diff
@@ -1,17 +1,11 @@
-{{#if showIndicators}}
-  <ol class='carousel-indicators'>
-    {{#each indicators as |indicator|}}
-      <li class='{{if indicator.active 'active'}}' onclick={{action 'toSlide' indicator.index}} role='button'></li>
-    {{/each}}
-  </ol>
-{{/if}}
+<ol class='carousel-indicators'>
```

Comment:
I think you still need to wrap this in the previous `{{#if}}` block, to not have a `<ol>` without any `<li>` if `showIndicators` is false.